src/app/packages/open-layers-north-calculations/plugin/north-calculations-plugin.ts

```diff
@@ -86,9 +86,9 @@ export class NorthCalculationsPlugin implements IMapPlugin {
 			// console.log(`setCorrectedNorth: (northOffsetDeg:'${northData.northOffsetDeg}')`);
 			if (Math.abs(northData.northOffsetDeg) < this.thresholdDegrees) {
 				// console.log(`setCorrectedNorth: (northOffsetDeg) break)`);
-				return;
+				return northData.actualNorth;
```

Comment:
can the function return typed answer (: Promise<number>), if it's to hard ignore this comment